### PR TITLE
zcs-911: Universal UI: Attach mail modal from compose view

### PR DESCRIPTION
**Pull request by tarangkhandelwal**
Changeset:
1. attachmail.css : Updated styles based on the new universal ui mockups.
2. attachmail.js :
	* Updated container positions for search and navigation as per new mockups.
	* Updated height of AttachMailTabView from 275 to 300 .

### CODE REVIEW

File: src/zimlet/com_zimbra_attachmail/attachmail.css [hunk 1]
```diff
@@ -28,35 +28,136 @@
...
+.AMZimletNavToolBarTitle {
+    height: 100%;
+    padding: 0 8px;
+}
+
+#attDlg_attMsg_SearchField {
+    width: 95%;
+    height: 3rem;
+    border:1px solid #E0E0E0;
+    height: 3rem;
```
Comment (kalpeshhpatel) [inline src/zimlet/com_zimbra_attachmail/attachmail.css]: Repeated `height` rule

File: src/zimlet/com_zimbra_attachmail/attachmail.css [hunk 2]
```diff
@@ -28,35 +28,136 @@
 }
 
 .AttachMailTabBox {
-    background-color: #FDFCFA;
-    border-right:1px solid #E0E0E0;
+    background-color: #ffffff;
```
Comment (SatyamKumarOnline) [inline src/zimlet/com_zimbra_attachmail/attachmail.css:31]: I believe, we donot use skin colours here, so we are going with this value ?

Comment (tarangkhandelwal) [reply src/zimlet/com_zimbra_attachmail/attachmail.css:31]: Yes skin framework does not apply to zimlet styles.

Comment (kalpeshhpatel) [reply src/zimlet/com_zimbra_attachmail/attachmail.css:31]: I think, yes, we can't use skin.properties variables here.
@tarangkhandelwal , let me know if I am wrong.

**Review by kalpeshhpatel — APPROVED** [review]

File: src/zimlet/com_zimbra_attachmail/attachmail.css [hunk 3]
```diff
@@ -28,35 +28,136 @@
...
+    border:1px solid #E0E0E0;
```
Comment (SatyamKumarOnline) [inline src/zimlet/com_zimbra_attachmail/attachmail.css]: Spacing. At other place, I find word gap for same format

File: src/zimlet/com_zimbra_attachmail/attachmail.css [hunk 4]
```diff
@@ -28,35 +28,136 @@
...
 
 .AttachMailSubject {
     font-weight: bold;
-    font-size: 13px;
-    color: #2B2A2B;
+    font-size: 12px;
+    color: rgb(51,51,51);
 }
 
 .AttachMailFrom {
```
Comment (SatyamKumarOnline) [inline src/zimlet/com_zimbra_attachmail/attachmail.css]: The css property of this class is same as of AttachMailSubject

File: src/zimlet/com_zimbra_attachmail/attachmail.css [hunk 5]
```diff
@@ -28,35 +28,136 @@
...
+.AttachMailList .Row-selected .AttachMailSubject {
+    color: #ffffff;
+}
+
+.Row-selected .svg-icon {
+    fill: #fff;
+}
+
+.AttachMailIcon,
+.AttachMailIcon > div{
```
Comment (SatyamKumarOnline) [inline src/zimlet/com_zimbra_attachmail/attachmail.css]: Spacing

File: src/zimlet/com_zimbra_attachmail/attachmail.css (hunk 4 above)
Comment (tarangkhandelwal) [reply src/zimlet/com_zimbra_attachmail/attachmail.css]: will merge them.

File: src/zimlet/com_zimbra_attachmail/attachmail.css [hunk 6]
```diff
@@ -28,35 +28,136 @@
...
+
+.AttachMailBottomRowRight {
+    width: 63%;
+    float: right;
+    margin-top: 3px;
+}
+
+.AMZimletNavToolBar .ZToolbarButton {
+    height: 36px;
+    height: 3rem;
```
Comment (SatyamKumarOnline) [inline src/zimlet/com_zimbra_attachmail/attachmail.css:109]: Repeated height value, one in px other in rem

File: src/zimlet/com_zimbra_attachmail/attachmail.css [hunk 7]
```diff
@@ -28,35 +28,136 @@
...
+
+.AMZimletNavToolBarTitle {
+    height: 100%;
+    padding: 0 8px;
+}
+
+#attDlg_attMsg_SearchField {
+    width: 95%;
+    height: 3rem;
+    border:1px solid #E0E0E0;
```
Comment (SatyamKumarOnline) [inline src/zimlet/com_zimbra_attachmail/attachmail.css]: spacing

File: src/zimlet/com_zimbra_attachmail/attachmail.css (hunk 6 above)
Comment (tarangkhandelwal) [reply src/zimlet/com_zimbra_attachmail/attachmail.css:109]: added just as a fallback. Existed in core zimbra fallback.

File: src/zimlet/com_zimbra_attachmail/attachmail.css [hunk 8]
```diff
@@ -28,35 +28,136 @@
...
+#attDlg_attMsg_ViewMsgBtn .ZHover .ZButtonBorder {
+    background: inherit;
+}
+
+#attDlg_attMsg_ViewMsgBtn .ZWidgetTitle  {
+    color: rgb(0, 124, 195);
+}
+
+#attDlg_attMsg_SearchBtn .ZButton {
+    border:1px solid #E0E0E0;
```
Comment (SatyamKumarOnline) [inline src/zimlet/com_zimbra_attachmail/attachmail.css]: spacing

File: src/zimlet/com_zimbra_attachmail/attachmail.css (hunk 3 above)
Comment (tarangkhandelwal) [reply src/zimlet/com_zimbra_attachmail/attachmail.css]: done

**Review by SatyamKumarOnline — APPROVED** [review]

**Review by silentsakky — APPROVED** [review]

**Pull request merged**